modules/ossm-cert-manager-verify.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+
+:_content-type: PROCEDURE
+[id="ossm-verify-cert-manager-installation_{context}"]
+= Verify cert-manager installation
```

Comment:
```suggestion
= Verifying cert-manager installation
```



Per mod docs procedure template, "Write a short introductory paragraph that provides an overview of the module. The introduction should include what the module will help the user do and why it will be beneficial to the user. Include key words that relate to the module to maximize search engine optimization."

This paragraph should be between the title and any prereqs (if they exist).